dev/tasks/build/builder.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* jshint node: true */
+
+'use strict';
+
+var Builder;
+
+/**
+ * A CKEditor 5 release builder.
+ *
+ * @class Builder
+ */
+module.exports = Builder = function( target ) {
```

Comment:
It's slightly better to use function declaration and then assign it to the `module.exports`, because it leaves the function named.